docs/pages/includes/role-spec.mdx

```diff
@@ -235,8 +235,15 @@ spec:
 
       # the `claims_to_roles` mapping works the same as it does in
       # the OIDC connector, with the added benefit that the roles being mapped to
-      # can also be matchers. the below mapping says that users with
-      # the claims `groups: admins` can request any role in the system.
+      # can also be matchers. 
+      #
+      # This example leverages Teleport's regular expression support, which allows 
+      # for dynamic mapping from claims. The below mapping says that users with
+      # claims that match "projects: project-(.*)" can request roles that match 
+      # "$1-admin", where "$1" is the first capture group in the 
+      # regular expression.
+      # Example: the "projects: project-foo" claim allows a user to request the
```

Comment:
To match the value below, would it be:

```suggestion
      # Example: the "projects: product-foo" claim allows a user to request the
```
?

Reply:
Thanks @alexfornuto , yes corrected.

Reply:
Please approve if that's good to go @alexfornuto